inference-engine/src/mkldnn_plugin/nodes/mkldnn_depth_to_space_node.cpp

```diff
@@ -58,6 +58,8 @@ MKLDNNDepthToSpaceNode::MKLDNNDepthToSpaceNode(const std::shared_ptr<ngraph::Nod
         if (blockSize == 0)
             THROW_ERROR << "has incorrect block_size parameter is zero!";
 
+        size_t nSpatialDims = inDims[0].size() - 2;
```

Comment:
MKLDNNDims::size doesn't return rank. You need ndims()

Reply:
fixed